### PR TITLE
Desktop: Linux: Prevent Desktop Environments to launch a new window

### PR DESCRIPTION
**Pull request by FHEK789**
X-GNOME-SingleWindow was upstreamed to be an XDG spec with the name
"SingleMainWindow" in
https://gitlab.freedesktop.org/xdg/xdg-specs/-/merge_requests/53.

This prevents Desktop Environments such as KDE and Gnome
to not display the menu item to open in a new window in apps that
don't support that.

### CODE REVIEW

**Comment by CalebJohn** [comment]
Thanks for the update @FHEK789, I think you might be a little early on this given that the change was just merged into Gnome/KDE 3 days ago :)
Do you think it would be safe to include both keys for now, and include a comment on the old way given some details that it can be removed in the future?

**Comment by FHEK789** [comment]
Included both keys and added an comment :)

**Comment by laurent22** [comment]
@FHEK789, could you give a less technical title to the issue please? This will end up in the release notes and I think most people won't know what it means. The title should explain in plain English what the changes - in this particular case I think it's meant to remove a menu item somewhere, is that correct?

**Comment by FHEK789** [comment]
Done!

**Comment by laurent22** [comment]
Looks good, thank you!
